search-app/src/cmr/search/site/static.clj

```diff
@@ -2,33 +2,68 @@
   "The functions of this namespace are specifically responsible for returning
   ready-to-serve pages."
   (:require
+   [cheshire.core :as json]
+   [clj-http.client :as client]
    [cmr.common-app.static :as static]
-   [cmr.search.site.data :as data])
+   [cmr.common.log :refer :all]
+   [cmr.search.site.data :as data]
+   [cmr.search.site.static.directory :as directory]
+   [cmr.search.site.static.site :as site]
+   [cmr.search.site.util :as util]
+   [cmr.transmit.config :as transmit])
   (:gen-class))
 
-(defn generate-api-docs
+;; Contextual data that is used for static content in the absense of a system
+;; context, e.g., when run from the command line.
+(defrecord StaticContext [
+  ;; the deployment env; one of :prod, :uat, :sit, or :local
+  deployment-target
```

Comment:
Is deployment-target being used anywhere?

Reply:
No; we decided not to go that route. I can remove.

Reply:
Removed.